src/App.tsx

```diff
@@ -65,7 +65,7 @@ export const App: React.FC = () => {
         <Header
           onTodoTitle={todoTitle}
           onSetTodoTitle={setTodoTitile}
-          onHandleSubmit={() => handleSubmit}
+          onHandleSubmit={handleSubmit}
```

Comment:
preposition 'on' just doesn't feel intuitively right
```suggestion
          todoTitle={todoTitle}
          setTodoTitle={setTodoTitile}
          handleSubmit={handleSubmit}
```